src/containers/Welcome/index.js

```diff
@@ -46,18 +52,27 @@ class Welcome extends React.Component {
     return (
       <RootView>
         <BottomAnchoredBackgroundImage />
-        <View style={{ flex: 1 }}>
-          <BrandingIcon />
-          <Heading>Welcome to MojoTech.</Heading>
-          <SubHeading>
-            Sign in to let your host know you&apos;re here.
-          </SubHeading>
-          <NameInputView
-            guestName={guestName}
-            handleNextPress={this.handleNextPress(guestName)}
-            handleChangeNameText={this.handleChangeNameText}
-          />
-        </View>
+        <KeyboardAvoidingView
+          behavior="position"
+          keyboardVerticalOffset={160}
+          enabled
+        >
+          <TouchableWithoutFeedback onPress={Keyboard.dismiss}>
+            <View>
+              <BrandingIcon />
+              <Heading>Welcome to MojoTech.</Heading>
+              <SubHeading>
+                Sign in to let your host know you&apos;re here.
+              </SubHeading>
+              <NameInputView
+                guestName={guestName}
+                handleNextPress={this.handleNextPress(guestName)}
+                handleChangeNameText={this.handleChangeNameText}
+                offsetOnFocus={-30}
```

Comment:
When the keyboard is activated `KeyboardAvoidingView` causes everything to scroll up. In addition to that, this offset makes the input move up a bit, closing the gap between the welcome text and the input. The shift is subtle, but it makes it so that the input looks more centered on the screen when the keyboard is both present and hidden. It can probably be removed, if the extra complexity is not desired.

Reply:
Strange. Removing it is probably a css rabbit hole. Best to leave it be  for now.